doc/docs/getting-started.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Getting Started
+slug: /
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Install
+
+<Tabs
+  groupId="lang"
+  defaultValue="hooks"
+  values={[
+    {label: 'React Hooks', value: 'hooks'},
+    {label: 'JavaScript', value: 'js'},
+  ]}>
+  <TabItem value="hooks">
+
+```bash
+npm add replicache replicache-react-util
+```
+
+  </TabItem>
+  <TabItem value="js">
+
+```bash
+npm add replicache
+```
+
+  </TabItem>
+</Tabs>
+
+## Serve Wasm Module
+
+Replicache uses [Wasm](https://webassembly.org/) internally. Most bundlers don't deal with this well yet,
+so it's easiest to just serve it as a static asset.
+
+Copy the Replicache Wasm module to your static assets directory:
+
+```bash
+cp node_modules/replicache/out/*.wasm public/
+```
+
+:::note
+
+You should add a [postinstall](https://docs.npmjs.com/cli/v7/using-npm/scripts) script to do this in your app, so that it will stay up to date when you update Replicache.
+
+:::
+
+## Client Setup
+
+<Tabs
+  groupId="lang"
+  defaultValue="hooks"
+  values={[
+    {label: 'React Hooks', value: 'hooks'},
+    {label: 'JavaScript', value: 'js'},
+  ]}>
+  <TabItem value="hooks">
+
+```ts
+import {Replicache} from "replicache";
+import {useSubscribe} from "replicache-react-util";
+
+const rep = new Replicache({
+  // Put the correct path to replicache.wasm.br on your server here.
+  wasmModule: "/replicache.wasm",
+
+  mutators: {
+    createTodo: (tx, args) => {
+      tx.put(`/todo/${args.id}`, args);
+    },
+  },
+});
+
+function MyComponent() {
+  const todos = useSubscribe(rep,
+    tx => tx.scan({prefix: "/todo/"}).toArray(), []);
+
+  const handleClick = () => {
+    rep.mutate.createTodo({
```

Comment:
missing await

We do want to make it clear that all of this is async

Reply:
Why? It doesn't make any difference to anything to make it async and to await it.